### ISSUE TITLE
[a11y] 9.1.4.11 - Sufficient contrast of graphics and graphical picture elements

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5386]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------

## General notes
- **Note:** information conveying graphics and graphical control elements as well as their states should have a contrast ration of at least 3:1 to their bordering colors.
- **Pages 1 to 3:** all input fields and check boxes have insufficient contrast ration when not focussed between the color of the element (border) and the background color - lower than 3:1.

## Status: rather not fulfilled
- **Page 1 (Alle Dateien)**
When creating a public link in the case the user doesn’t fill the fields a error message appears above the form. This error message has a „X“ button on the right (to remove the message). Here the contrast ration to the background color is lower than 3:1.
See general notes

- **Page 2 (Mit anderen geteilten Dateien)**
See general notes
- **Page 3 (Gelöschte Dateien)**
See general notes

## Status: fulfilled
- **Page 4 (Konto)**

**Comment by pascalwengerter** [comment]
Superseded by the more recent #10729
